docs/docs/development/backend/authentication.md

```diff
@@ -3,15 +3,20 @@
  ~ SPDX-License-Identifier: Apache-2.0
  -->
 
-## Authentication via Personal Access Token (Requires a Running Frontend)
+The are two common ways to authenticate against the backend in a development
```

Comment:
Not sure you need "common" here, there are actually only 2 ways to authenticate here, right?

Reply:
Not needed, removed it.